### PR TITLE
Add compat data for api.Navigator

### PR DESCRIPTION
**Pull request by luciancic**
[Navigator](https://developer.mozilla.org/en-US/docs/Web/API/Navigator)
Done

### CODE REVIEW

File: api/Navigator.json [hunk 1]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "mozNotification": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/mozNotification",
```
Comment (maboa) [inline api/Navigator.json]: Not seeing a table for this.

File: api/Navigator.json [hunk 2]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "mozSocial": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/mozSocial",
```
Comment (maboa) [inline api/Navigator.json]: Not seeing a page or table for this.

File: api/Navigator.json [hunk 3]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "standalone": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/standalone",
```
Comment (maboa) [inline api/Navigator.json]: Not seeing a page or table for this.

File: api/Navigator.json [hunk 4]
```diff
@@ -277,66 +478,1757 @@
...
+                "version_added": true
+              },
+              "safari_ios": {
+                "version_added": true
+              },
+              "samsunginternet_android": {
+                "version_added": null
+              },
+              "webview_android": {
+                "version_added": null
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Not the same as webview

File: api/Navigator.json [hunk 5]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "mediaSession": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/mediaSession",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Added in [PR 1533](https://github.com/mdn/browser-compat-data/pull/1533)

File: api/Navigator.json (hunk 1 above)
Comment (lucian95) [reply api/Navigator.json]: Fair enough. Shall I remove it? I think it should be obvious to the reader that mozNotification is not supported on other vendors (but maybe not).

File: api/Navigator.json [hunk 6]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "mozPay": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/mozPay",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: same

File: api/Navigator.json (hunk 2 above)
Comment (lucian95) [reply api/Navigator.json]: same

File: api/Navigator.json [hunk 7]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": true
+          }
+        }
+      },
+      "oscpu": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/oscpu",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Did tests on my own browsers

File: api/Navigator.json [hunk 8]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": true,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "presentation": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/presentation",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: same

File: api/Navigator.json [hunk 9]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "productSub": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/productSub",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Went with notes from page and own browser tests

File: api/Navigator.json [hunk 10]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "securitypolicy": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/securitypolicy",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: There was a reason I put it there but I can't remember. I will remove it in the meanwhile.

File: api/Navigator.json (hunk 3 above)
Comment (lucian95) [reply api/Navigator.json]: same

File: api/Navigator.json [hunk 11]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "storageQuota": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/storageQuota",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Removed

File: api/Navigator.json [hunk 12]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "vendor": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/vendor",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Removed

File: api/Navigator.json [hunk 13]
```diff
@@ -277,66 +478,1757 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/vendorSub",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Made poor assumption, changed it now

File: api/Navigator.json [hunk 14]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "webkitNotification": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/webkitNotification",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Removed

File: api/Navigator.json [hunk 15]
```diff
@@ -277,66 +478,1757 @@
...
+          "status": {
+            "experimental": false,
+            "standard_track": false,
+            "deprecated": false
+          }
+        }
+      },
+      "webkitPointer": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Navigator/webkitPointer",
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Removed

File: api/Navigator.json [hunk 16]
```diff
@@ -277,66 +478,1757 @@
...
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
```
[the remark this answers is not in the document]
Comment (lucian95) [reply api/Navigator.json]: Linter output: 
```  
  version_added: "12" is not a valid version number for ie
  Valid ie versions are: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 1.5, 5.5
  File : /home/me/dev/browser-compat-data/api/Navigator.json
```
So I decided to go with `true`

**Comment by connorshea** [comment]
I think this is missing [`getGamepads()`](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/getGamepads)

**Comment by connorshea** [comment]
I opened #2423 to add `getGamepads()`.

**Comment by connorshea** [comment]
This also missed [maxTouchPoints](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/maxTouchPoints), though that API's compat table is pretty weird.

It looks like it was enabled by default in Firefox 59 on desktop https://bugzilla.mozilla.org/show_bug.cgi?id=1411467, but it hasn't been enabled by default on Firefox for Android yet: https://bugzilla.mozilla.org/show_bug.cgi?id=1426786

**Comment by connorshea** [comment]
Opened #2508 to add `maxTouchPoints`.

**Pull request merged**